### PR TITLE
Add femboy armor as alternate textures for the Manaweave armor

### PR DESCRIPTION
**Pull request by gamma-delta**
I'm not sure who originally suggested it but whoever did you're extremely cursed >:(

Also, this removes the "old" armor variants and renames the `_NEW` armors to just the armors. (`{ARMOR}_MANASTEEL_NEW` -> `{ARMOR}_MANASTEEL`)

Get the armor by renaming the Manaweave set in an anvil to... you'll just have to figure it out ;)

### CODE REVIEW

**Comment by ayyydriannn** [comment]
this is beyond blursed

File: src/main/java/vazkii/botania/common/lib/LibMisc.java [hunk 1]
```diff
@@ -16,4 +16,21 @@
...
+		int string = 0x23;
+		while (n.length() < string) {
+			n += delegate;
+		}
+		String var = n.substring(0, 9);
+		String val = var.substring(0, 6);
+
+		int bool = 0x621;
+		for (int let = 1; let < (var + val).codePointBefore(val.length() * 2 - 1 - 3) << string - 0b10100 + delegate.charAt(delegate.length() - 1); let -= -string * let) {
+			bool ^= let == string ? var.hashCode() : (val.hashCode() / bool) % 2 == 0 ? string : var.hashCode() ^ val.hashCode() + bool;
```
Comment (Alwinfy) [inline src/main/java/vazkii/botania/common/lib/LibMisc.java:30]: Passing the string `Exarwtw's` to this function throws a divide by zero on this line

**Comment by Alwinfy** [comment]
After manually fuzzing based on the given hints, I have determined the following strings are false positives to `isUwu`:
`Cuyktaq's`, `Ednmjso's`, `Lsmdvzn's`, `Pkgoekd's`, `Qwyirmq's`, `Rxekrhn's`, `Slbalbm's`, `Syaekrq's`, `Tzyanha's`, `Uqpcumo's`, `Vfjkgxn's`, `Ybjaszo's`
Statistics says there are about 3.6e34 more of 'em

**Comment by williewillus** [comment]
this should probably belong in an addon

**Comment by Hubry** [comment]
make a cool addon with this instead :wacko:

**Pull request closed without merging**